server/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -191,6 +193,8 @@ public static HighlightBuilder highlight() {
 
     private PointInTimeBuilder pointInTimeBuilder = null;
 
+    private Map<String, Object> runtimeMappings = null;
```

Comment:
shall we initialize it with empty list and make sure that it can never be null, then we can remove the boolean flag in the serialization methods?

Reply:
I was using the pattern from stuff like `sort`, but I see that lots of stuff uses the emptyMap style. I'll swap.